Makefile

```diff
@@ -5,10 +5,12 @@ SHELL := bash
 
 # Set the package's name and version for use throughout the Makefile.
 PACKAGE_NAME := package
-ifeq ($(origin VIRTUAL_ENV),undefined)
-  PACKAGE_VERSION := unknown
-else
+ifneq ($(origin VIRTUAL_ENV),undefined)
   PACKAGE_VERSION := $(shell python -c 'import $(PACKAGE_NAME); print($(PACKAGE_NAME).__version__)')
+else ifneq ($(wildcard .venv/upgraded-on),)
+  PACKAGE_VERSION := $(shell .venv/bin/python -c 'import $(PACKAGE_NAME); print($(PACKAGE_NAME).__version__)')
```

Comment:
I guess if we use `.venv/bin/python` to determine the package version then we should use the [`$(wildcard)`](https://www.gnu.org/software/make/manual/html_node/Wildcard-Function.html) function to check for `python`, instead of the `.venv/upgraded-on` timestamp?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/387/commits/30641a024e2791a03080ce8aa0b3775f87178477.